Public/Src/Engine/Dll/Engine.cs

```diff
@@ -1066,6 +1066,7 @@ public static void PopulateLoggingAndLayoutConfiguration(CommandLineConfiguratio
                 }
 
                 mutableConfig.Logging.StoreFingerprints = initialCommandLineConfiguration.Logging.StoreFingerprints ?? false;
+                mutableConfig.Sandbox.RetryOnDeadExitCode = true;

```

Comment:
>Dead [](start = 45, length = 4)

Maybe rename this to `AzureWatsonExitCode` Now it reads there is a generic exit code when a process crashes.

Reply:
Note comment applies to all usage of 'dead' (other than the const that azure defined)

---
In reply to: [289928447](https://github.com/microsoft/BuildXL/pull/401#discussion_r289928447) [](ancestors = 289928447)